### PR TITLE
Line length

### PR DESCRIPTION
**Pull request by arschmitz**
This fixes all the maxLineLength jscs issues except in tests

### CODE REVIEW

File: ui/effect.js [hunk 1]
```diff
@@ -728,14 +734,15 @@ var classAnimationActions = [ "add", "remove", "toggle" ],
...
 
-$.each( [ "borderLeftStyle", "borderRightStyle", "borderBottomStyle", "borderTopStyle" ], function( _, prop ) {
-	$.fx.step[ prop ] = function( fx ) {
-		if ( fx.end !== "none" && !fx.setAttr || fx.pos === 1 && !fx.setAttr ) {
-			jQuery.style( fx.elem, prop, fx.end );
-			fx.setAttr = true;
-		}
-	};
-} );
+$.each( [ "borderLeftStyle", "borderRightStyle", "borderBottomStyle", "borderTopStyle" ],
```
Comment (scottgonzalez) [inline ui/effect.js]:
```js
$.each(
    [ ... ],
    function() {
        ...
    }
);

File: ui/effect.js [hunk 2]
```diff
@@ -1491,7 +1502,8 @@ $.fn.extend( {
 
 	cssClip: function( clipObj ) {
 		return clipObj ?
-			this.css( "clip", "rect(" + clipObj.top + "px " + clipObj.right + "px " + clipObj.bottom + "px " + clipObj.left + "px)" ) :
+			this.css( "clip", "rect(" + clipObj.top + "px " + clipObj.right + "px " +
+				clipObj.bottom + "px " + clipObj.left + "px)" ) :
 			parseClip( this.css( "clip" ), this );
```
Comment (scottgonzalez) [inline ui/effect.js]: Let's move away from the ternary here.

File: ui/widgets/draggable.js [hunk 3]
```diff
@@ -291,12 +295,19 @@ $.widget( "ui.draggable", $.ui.mouse, {
...
+				this.options.revert.call( this.element, dropped ) )
+		) {
+			$( this.helper ).animate(
+				this.originalPosition,
+				parseInt( this.options.revertDuration, 10 ),
+				function() {
+					if ( that._trigger( "stop", event ) !== false ) {
+						that._clear();
+					}
+				} );
```
Comment (scottgonzalez) [inline ui/widgets/draggable.js]: Move the closing parent to the next line.

File: ui/widgets/draggable.js [hunk 4]
```diff
@@ -486,8 +508,11 @@ $.widget( "ui.draggable", $.ui.mouse, {
 			this.containment = [
 				$( window ).scrollLeft() - this.offset.relative.left - this.offset.parent.left,
 				$( window ).scrollTop() - this.offset.relative.top - this.offset.parent.top,
-				$( window ).scrollLeft() + $( window ).width() - this.helperProportions.width - this.margins.left,
-				$( window ).scrollTop() + ( $( window ).height() || document.body.parentNode.scrollHeight ) - this.helperProportions.height - this.margins.top
+				$( window ).scrollLeft() + $( window ).width() - this.helperProportions.width -
```
Comment (scottgonzalez) [inline ui/widgets/draggable.js]: I'd split this on the first minus to group together the growing/shrinking pairs.

File: ui/widgets/draggable.js [hunk 5]
```diff
@@ -486,8 +508,11 @@ $.widget( "ui.draggable", $.ui.mouse, {
...
+					this.margins.left,
+				$( window ).scrollTop() + ( $( window ).height() ||
```
Comment (scottgonzalez) [inline ui/widgets/draggable.js]: I'd put the grouped section onto its own line.

File: ui/widgets/draggable.js [hunk 6]
```diff
@@ -969,16 +1048,20 @@ $.ui.plugin.add( "draggable", "scroll", {
...
 				}
 			}
 
 			if ( !o.axis || o.axis !== "y" ) {
 				if ( event.pageX - $( document ).scrollLeft() < o.scrollSensitivity ) {
-					scrolled = $( document ).scrollLeft( $( document ).scrollLeft() - o.scrollSpeed );
-				} else if ( $( window ).width() - ( event.pageX - $( document ).scrollLeft() ) < o.scrollSensitivity ) {
-					scrolled = $( document ).scrollLeft( $( document ).scrollLeft() + o.scrollSpeed );
+					scrolled =
+						$( document ).scrollLeft( $( document ).scrollLeft() - o.scrollSpeed );
```
Comment (scottgonzalez) [inline ui/widgets/draggable.js]:
```js
scrolled = $( document ).scrollLeft(
    $( document ).scrollLeft() - o.scrollSpeed
);
```

Same a few lines down.

File: ui/widgets/droppable.js [hunk 7]
```diff
@@ -178,25 +182,37 @@ $.widget( "ui.droppable", {
...
+			.not( ".ui-draggable-dragging" )
+			.each( function() {
+				var inst = $( this ).droppable( "instance" );
+				if (
+					inst.options.greedy &&
+					!inst.options.disabled &&
+					inst.options.scope === draggable.options.scope &&
+					inst.accept
+						.call(
+							inst.element[ 0 ], ( draggable.currentItem || draggable.element ) ) &&
```
Comment (scottgonzalez) [inline ui/widgets/droppable.js]: Drop the closing paren to the next line.

File: ui/widgets/droppable.js [hunk 8]
```diff
@@ -178,25 +182,37 @@ $.widget( "ui.droppable", {
...
+					intersect(
+						draggable,
+						$.extend( inst, { offset: inst.element.offset() } ),
+						inst.options.tolerance, event )
```
Comment (scottgonzalez) [inline ui/widgets/droppable.js]: Same here.

File: ui/widgets/menu.js [hunk 9]
```diff
@@ -142,7 +144,8 @@ return $.widget( "ui.menu", {
 			},
 			blur: function( event ) {
 				this._delay( function() {
-					if ( !$.contains( this.element[ 0 ], $.ui.safeActiveElement( this.document[ 0 ] ) ) ) {
+					if ( !$.contains( this.element[ 0 ],
```
Comment (scottgonzalez) [inline ui/widgets/menu.js]: Let's move this contains call out of the conditional and store it in a variable, then put the variable in the conditional. It'll allow much nicer linebreaks.

File: ui/widgets/menu.js [hunk 10]
```diff
@@ -92,7 +92,9 @@ return $.widget( "ui.menu", {
 					// Open submenu on click
 					if ( target.has( ".ui-menu" ).length ) {
 						this.expand( event );
-					} else if ( !this.element.is( ":focus" ) && $( $.ui.safeActiveElement( this.document[ 0 ] ) ).closest( ".ui-menu" ).length ) {
+					} else if ( !this.element.is( ":focus" ) &&
+							$( $.ui.safeActiveElement( this.document[ 0 ] ) )
+							.closest( ".ui-menu" ).length ) {
```
Comment (scottgonzalez) [inline ui/widgets/menu.js]: This line needs to be indented one more level, but that gets really crazy. Let's store the active element in a var to reduce the save.

File: ui/widgets/sortable.js [hunk 11]
```diff
@@ -331,32 +345,43 @@ return $.widget( "ui.sortable", $.ui.mouse, {
...
+						o.scrollSensitivity ) {
 					scrolled = this.document.scrollTop( this.document.scrollTop() + o.scrollSpeed );
 				}
 
 				if ( event.pageX - this.document.scrollLeft() < o.scrollSensitivity ) {
-					scrolled = this.document.scrollLeft( this.document.scrollLeft() - o.scrollSpeed );
-				} else if ( this.window.width() - ( event.pageX - this.document.scrollLeft() ) < o.scrollSensitivity ) {
-					scrolled = this.document.scrollLeft( this.document.scrollLeft() + o.scrollSpeed );
+					scrolled =
+						this.document.scrollLeft( this.document.scrollLeft() - o.scrollSpeed );
```
Comment (scottgonzalez) [inline ui/widgets/sortable.js]:
```js
scrolled = this.document.scrollLeft(
    this.document.scrollLeft() - o.scrollSpeed
);
```

Same a few lines down.

File: ui/widgets/sortable.js [hunk 12]
```diff
@@ -455,15 +483,26 @@ return $.widget( "ui.sortable", $.ui.mouse, {
...
 			this.reverting = true;
-			$( this.helper ).animate( animation, parseInt( this.options.revert, 10 ) || 500, function() {
-				that._clear( event );
-			} );
+			$( this.helper ).animate(
+					animation,
+					parseInt( this.options.revert, 10 ) || 500,
+					function() {
+						that._clear( event );
+					} );
```
Comment (scottgonzalez) [inline ui/widgets/sortable.js]: This whole block is over-indented. Also move the closing paren to the next line.

**Comment by jzaefferer** [comment]
@arschmitz let's finish this! should go into the rc2 release.

**Comment by arschmitz** [comment]
updated

File: ui/effect.js [hunk 13]
```diff
@@ -728,14 +734,16 @@ var classAnimationActions = [ "add", "remove", "toggle" ],
...
+$.each(
+	[ "borderLeftStyle", "borderRightStyle", "borderBottomStyle", "borderTopStyle" ],
+	function( _, prop ) {
+		$.fx.step[ prop ] = function( fx ) {
+			if ( fx.end !== "none" && !fx.setAttr || fx.pos === 1 && !fx.setAttr ) {
+				jQuery.style( fx.elem, prop, fx.end );
+				fx.setAttr = true;
+			}
+		};
+	} );
```
Comment (scottgonzalez) [inline ui/effect.js]: Move the closing paren to the next line.

File: ui/widgets/droppable.js [hunk 14]
```diff
@@ -178,25 +182,39 @@ $.widget( "ui.droppable", {
...
+			.find( ":data(ui-droppable)" )
+			.not( ".ui-draggable-dragging" )
+			.each( function() {
+				var inst = $( this ).droppable( "instance" );
+				if (
+					inst.options.greedy &&
+					!inst.options.disabled &&
+					inst.options.scope === draggable.options.scope &&
+					inst.accept
+						.call(
```
Comment (scottgonzalez) [inline ui/widgets/droppable.js]: I'd move this up to the previous line, then outdent the next two lines.

**Pull request closed without merging**